### ISSUE TITLE
[Issue]: NFO defined backdrops result in addition duplicates after each refresh all images

### ISSUE THREAD

**Issue by SenorSmartyPants** [jellyfin/jellyfin#9456]
### Please describe your bug

I have an NFO for a TV season. I'm setting a backdrop using the NFO, remote image providers have no backdrops for this season. Every time I I run refresh metadata (replace all images) I get another backdrop of the same image from the NFO. I have no local images in the season directory, and JF does not create any there. Sounds like #7836 but that is reportedly fixed. I just tested against release-10.8.z branch

I think the root of the problem is here
https://github.com/jellyfin/jellyfin/blob/6821a2ab358761282a0030c42c837b39bad089e1/MediaBrowser.Providers/Manager/MetadataService.cs#L702
localItem.Images never has any images. Images are being added directly to item a few lines above. But I can't figure out what a fix should be

```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<season>
	<thumb aspect="poster">
		https://thetvdb.com/banners/seasons/72306-2-5.jpg
	</thumb>
	<fanart>
		<thumb>
			https://artworks.thetvdb.com/banners/fanart/original/72306-8.jpg
		</thumb>
	</fanart> 
</season>
```

### Jellyfin Version

10.8.0

### if other:

10.8.z release branch compiled locally for debugging

### Environment

```markdown
- OS:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Please attach any screenshots here

![image](https://user-images.githubusercontent.com/991618/223863432-d21f8bf8-2618-4825-a654-ef96f7510b97.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Shadowghost** [comment]
Is this specific to seasons? I tried replicating it on current master with a series and I was not able to do so. I did find series with multiple copies of the same backdrop though

**Comment by SenorSmartyPants** [comment]
Yes, backdrop has different logic because it is allowed to have more than 1 image for that type.

**Comment by SenorSmartyPants** [comment]
Don't close this.

**Comment by SenorSmartyPants** [comment]
Still an issue. There is a PR to fix, waiting to be merged.

**Comment by SenorSmartyPants** [comment]
Still an issue. There is a PR to fix, waiting to be merged.

**Comment by theguymadmax** [comment]
This issue can no longer be reproduced, closing as fixed
